django_project_491/django_app/models.py

```diff
@@ -127,7 +127,7 @@ def run_snippet(self): # TODO
         else:
             return result['stdout'].split('\n')
 
-    def mark_as_answered(self): # TODO
+    def mark_as_answered(self, comment_id): # TODO
```

Comment:
why did you add comment_id here? Can a user choose the comment as the answer to a question? In any case, I think this code is incomplete since you do not use comment id inside the function, please implement it so that we will not forget about it later.

Reply:
The reason behind my comment_id parameter addition is that the usage of the mark_as_anwered method in [comment_views.py](https://github.com/bounswe/bounswe2024group9/blob/main/django_project_491/django_app/views/comment_views.py). Since this page was created ahead of the models.py I took the views class as reference.